### DISCUSSION TITLE
Can AIR make a screen companion game for windows?

### DISCUSSION THREAD

**Discussion by GeneralVimes** [airsdk/Adobe-Runtime-Support#3217] [General]
Today I saw this game: https://store.steampowered.com/app/2666510/Rustys_Retirement/
Its window is located at the bottom of the screen, allowing the player to spend time on other tasks while keeping track on what’s happening in the game.

I wonder, is this effect achievable with AIR for Windows and under what settings?

**Comment by ajwfrost** [comment]
Do you know whether this is something that's affecting the desktop size i.e. other apps automatically snap to that when maximized, or do they snap to the normal taskbar and this just runs over the top? If the latter, it should be possible currently, the only issue perhaps being whether you can position it properly above the task bar. Otherwise though, it may need an ANE to get the Windows desktop to set up properly with this running in such a fashion...

thanks

**Reply by GeneralVimes** [reply]
Thank you, I will check how this works in details and we'll see

**Reply by dyxribo** [reply]
can you set the native window chrome to `NativeWindowSystemChrome.NONE` and use `NativeWindowBoundsEvent.RESIZING` and `NativeWindowDisplayStateEvent.DISPLAY_STATE_CHANGE` events to listen for snap and manual resizes? that might allow exactly what you're looking for, your game would just be responsible for updating visuals when the events are dispatched.
